public/stylesheets/home.css

```diff
@@ -0,0 +1,202 @@
+/* 88888888888888888    slider   8888888888888888888888 */
```

Comment:
Don't checkin the public/* it should be git ignore

Reply:
Done